partner_shipper/README.rst

```diff
@@ -0,0 +1,103 @@
+=====================
```

Comment:
Please delete this README.rst file
As we knows, the readme generator is not working outside OCA.